pkg/network/usm/ebpf_javatls.go

```diff
@@ -99,7 +99,6 @@ var javaTLSSpec = &protocols.ProtocolSpec{
 				EBPFFuncName: doVfsIoctlKprobeName,
 				UID:          probeUID,
 			},
-			KProbeMaxActive: maxActive,
```

Comment:
setting this to const value can actually decrease the [default size](https://elixir.bootlin.com/linux/latest/source/kernel/kprobes.c#L2227) (consider a host with a large number of CPUs).

maybe switch to `max(num_of_cpus*2, max_active)`?

P.S: in older kernel versions the maxactive was [set](https://elixir.bootlin.com/linux/v5.2/source/kernel/kprobes.c#L1908) based on the `[CONFIG_PREEMPT](https://elixir.bootlin.com/linux/v5.2/K/ident/CONFIG_PREEMPT)` flag

Reply:
we can track (and compare in a load-test) the `nmissed` field of that kprobe. I'll approve this PR, as i agree with "currently it feels as an overkill".

But do consider making that test